server/schemas/shop.js

```diff
@@ -0,0 +1,25 @@
+let mongoose = require("mongoose");
+
+const itemSchema = new mongoose.Schema({
+  Title: { type: String },
+  price: { type: Number },
+  description: { type: String },
+  category: { type: String },
+  photos: [{ type: String }]
+});
+
+let shopSchema = new mongoose.Schema({
+  user: { type: mongoose.Schema.Types.ObjectId, ref: "users" },
+  title: {
+    type: String
+  },
+  description: {
+    type: String
+  },
+  cover_photo: {
+    type: String
+  },
+  items: [itemSchema]
```

Comment:
Do we want this to be in a separate model like we did between shop and user, or nested like above ^ It would depend on how many items we expect in each shop. I'm fine with either approach

Reply:
Each shop will have items at one point as this makes more sense to me.